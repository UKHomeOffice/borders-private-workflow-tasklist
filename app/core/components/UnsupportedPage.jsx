import React from 'react';

const UnsupportedPage = () => (
  <div
    style={{ marginTop: '20px' }}
    className="govuk-error-summary"
    aria-labelledby="error-summary-title"
    role="alert"
    tabIndex="-1"
    data-module="error-summary"
  >
    <h2 className="govuk-error-summary__title" id="error-summary-title">
      Sorry, your browser is not supported
    </h2>
    <div className="govuk-error-summary__body">
      <p>
        Please use one of the{' '}
        <a
          href="https://doc.dev.cop.homeoffice.gov.uk/service.html#operational-support-model"
          target="_blank"
          rel="noopener noreferrer"
        >
          supported browsers
        </a>
        .
      </p>
      <p>
        Contact the{' '}
        <a href="https://support.cop.homeoffice.gov.uk/servicedesk/customer/portal/3/">
          COP Service Support Desk
        </a>{' '}
        if you have any questions.
      </p>
      <h3
        className="govuk-heading-s"
      >
        Support hours
      </h3>
      <p>Monday to Friday: 8am to 6pm </p>
    </div>
  </div>
);

export default UnsupportedPage;
