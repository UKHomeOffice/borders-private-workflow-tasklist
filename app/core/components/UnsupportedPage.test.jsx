import React from 'react';
import UnsupportedPage from './UnsupportedPage';

describe('UnsupportedPage page', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<UnsupportedPage />);
    expect(wrapper.exists()).toBe(true);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<UnsupportedPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
