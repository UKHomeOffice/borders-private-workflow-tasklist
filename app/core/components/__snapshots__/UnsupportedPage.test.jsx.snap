// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnsupportedPage page matches snapshot 1`] = `
<div
  aria-labelledby="error-summary-title"
  className="govuk-error-summary"
  data-module="error-summary"
  role="alert"
  style={
    Object {
      "marginTop": "20px",
    }
  }
  tabIndex="-1"
>
  <h2
    className="govuk-error-summary__title"
    id="error-summary-title"
  >
    Sorry, your browser is not supported
  </h2>
  <div
    className="govuk-error-summary__body"
  >
    <p>
      Please use one of the
       
      <a
        href="https://doc.dev.cop.homeoffice.gov.uk/service.html#operational-support-model"
        rel="noopener noreferrer"
        target="_blank"
      >
        supported browsers
      </a>
      .
    </p>
    <p>
      Contact the
       
      <a
        href="https://support.cop.homeoffice.gov.uk/servicedesk/customer/portal/3/"
      >
        COP Service Support Desk
      </a>
       
      if you have any questions.
    </p>
    <h3
      className="govuk-heading-s"
    >
      Support hours
    </h3>
    <p>
      Monday to Friday: 8am to 6pm 
    </p>
  </div>
</div>
`;
